src/sql/workbench/contrib/welcome/gettingStarted/browser/initialSetupWizard.ts

```diff
@@ -0,0 +1,452 @@
+		popups[i].classList.remove('ads-initial-setup-wizard-show');
+		popups[last].classList.add('ads-initial-setup-wizard-show');
+		popups[last].classList.remove('ads-initial-setup-wizard-hide');
+		h1.focus();
+	}
+
+
+	private buildInteractions(): void {
+		const popups = document.querySelectorAll('.ads-initial-setup-wizard-popup') as NodeListOf<Element>;
+		const menuBarItems = document.querySelectorAll('.menubar-menu-button') as NodeListOf<HTMLElement>;
+		const context: this = this;
+		menuBarItems.forEach((elm) => {
+			elm.style.pointerEvents = 'none';
+		});
+		popups.forEach(function (elm, i) {
+			const nextBtn = elm.querySelector('.ads-initial-setup-wizard-next-btn-container .monaco-button') as HTMLElement;
+			const previousBtn = elm.querySelector('.ads-initial-setup-wizard-previous-btn-container .monaco-button') as HTMLElement;
+			const popupsLength: number = popups.length;
+
+			dom.addStandardDisposableListener(nextBtn, 'keydown', event => {
+				if (event.equals(KeyCode.Enter)) {
+					context.wizardNextEvent(popups, popupsLength, i);
+				}
+				if (event.equals(KeyCode.Tab)) {
+					event.preventDefault();
+				}
+			});
+
+			dom.addStandardDisposableListener(nextBtn, 'click', event => {
+				context.wizardNextEvent(popups, popupsLength, i);
+			});
+
+			if (previousBtn) {
+
+				dom.addStandardDisposableListener(previousBtn, 'keydown', event => {
+					if (event.equals(KeyCode.Enter)) {
+						context.wizardPrevEvent(popups, popupsLength, i);
+					}
+				});
+
+				dom.addStandardDisposableListener(previousBtn, 'click', event => {
+					context.wizardPrevEvent(popups, popupsLength, i);
+				});
+			}
+		});
+		this.show();
+	}
+
+	public show(): void {
+		if (this._overlay.style.display !== 'block') {
+
+			const firstSetupElement = document.querySelector('.ads-initial-setup-wizard-element') as HTMLDivElement;
+			const setupWizard = document.querySelector('.ads-initial-setup-wizard') as HTMLElement;
+
+			setTimeout(() => {
+				setupWizard.classList.add('open');
+				firstSetupElement.classList.add('ads-initial-setup-animate');
+				firstSetupElement.classList.add('ads-initial-setup-wizard-show');
+			}, 1000);
+
+
+			this._overlay.focus();
+		}
+	}
+
+	public hide(): void {
+		const menuBarItems = document.querySelectorAll('.menubar-menu-button') as NodeListOf<HTMLElement>;
+		const previewFeaturesOptIn = document.querySelector(`input[name='previewFeatures']`) as HTMLInputElement;
+		const setupWizaredContainer = document.querySelector('.ads-initial-setup-wizard') as HTMLDivElement;
+		if (previewFeaturesOptIn.checked) {
+			this.configurationService.updateValue('workbench.enablePreviewFeatures', true).catch(e => onUnexpectedError(e));
+			this.storageService.store(GettingStartedSetupWizard.ENABLE_PREVIEW_FEATURES_SHOWN, true, StorageScope.GLOBAL);
+		}
+		menuBarItems.forEach(function (elm) {
+			elm.style.pointerEvents = 'auto';
+		});
+		initialSetupWizardData.forEach(function ({ id }) {
```

Comment:
What is this removing? Wouldn't removing the container also remove all the elements created?